test/core/Do.Test.Architecture/Architecture/Feature/ConfiguringLayers.cs

```diff
@@ -21,6 +21,21 @@ public void Configure(LayerConfigurator configurator)
         }
     }
 
+    protected class Feature : IFeature
+    {
+        public void Configure(LayerConfigurator configurator) =>
+            configurator.Configure((ConfigurationA configuration) =>
+                configuration.Value = "Test");
```

Comment:
this configuration is unnecessary, don't create stubs that you don't use in test, this method should have an empty body